docker-compose.override.yml

```diff
@@ -89,6 +95,7 @@ services:
   mssql-init:
     isolation: ${ISOLATION}
     image: ${MVP_DOCKER_REGISTRY}mvp-mssql-init:${SITECORE_VERSION}
+    restart: ${DOCKER_RESTART}
```

Comment:
Do we want to restart the MSSQL Init container job? 

This should terminate once the job is complete.